bitnami/kafka/templates/kafka-provisioning.yaml

```diff
@@ -0,0 +1,91 @@
+{{- if .Values.provisioning.enabled }}
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: {{ include "kafka.provisioning.fullname" . }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: kafka
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+spec:
+  template:
+    metadata:
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: kafka
+        {{- if .Values.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+      annotations:
+    spec:
+{{- include "kafka.imagePullSecrets" . | indent 6 }}
+      restartPolicy: OnFailure
+      terminationGracePeriodSeconds: 0
+      containers:
+        - name: kafka-provisioning
+          image: {{ include "kafka.provisioning.image" . }}
+          imagePullPolicy: {{ .Values.provisioning.image.pullPolicy | quote }}
+          command:
+            - /bin/bash
+            - -c
+            - >-
+            {{- $bootstrapServer := printf "%s:%d" (include "kafka.fullname" .) (.Values.service.port | int64) }}
```

Comment:
Would it make sense to add a wait for the server to be ready? We have a binary called wait-for-port that could work for this case, but we would need to add it to the container.

Reply:
I tested this problem, kafka-topics retries if kafka is not ready without exiting the process. But i think adding wait-for-port as init container still a good idea

Reply:
Because if kafka is not running or start slowly kafka-topics will throw

Reply:
@javsalgar I've not found docker image of bitnami wait-for-port in dockerhub, can you help me?

I found Dockerfile in repo, but i think ci is not publishing it

Reply:
for the retry just use restart OnFailure..

Reply:
Hmm? I'm already using it

Reply:
Hi,
  `wait-for-port` is a small binary included in some of our container, for example [here](https://github.com/bitnami/bitnami-docker-zookeeper/blob/8ea99b75582c9565edaad7b499a397fd7dfd9f5d/3/debian-10/Dockerfile#L15)

Reply:
@rafariossaa https://github.com/bitnami/bitnami-docker-kafka/pull/135

Reply:
Hi, 
  We had released a kafka image that includes that change.

Reply:
Yes, i added wait-for-port as initContainer to job